tests/phpunit/test-main.php

```diff
@@ -14,7 +14,7 @@ class Test_Main extends WP_UnitTestCase {
 	 */
 	public function test_mu_plugin_constants() {
 		$this->assertTrue( defined( 'PANTHEON_MU_PLUGIN_VERSION' ) );
-		$this->assertEquals( '1.2.1', PANTHEON_MU_PLUGIN_VERSION );
+		$this->assertEquals( '1.3.1', PANTHEON_MU_PLUGIN_VERSION );
```

Comment:
this seems like a useless test 🤷🏻 IMO isDefined seems fine.

Reply:
Yeah that's fair. I can pull it out. Maybe someday we could write a test that matches it against the gh release or something...